### PR TITLE
Contributing Guide check-in

### PR DESCRIPTION
**Pull request by gabeah**
Ready for overview by supervisors

### CODE REVIEW

File: test/generate-inputs/expected/local_neighborhood-network-expected.txt.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+A|B
```
Comment (annaritz) [inline test/generate-inputs/expected/local_neighborhood-network-expected.txt.txt:1]: remove one of the `.txt`s

**Review by annaritz — CHANGES_REQUESTED** [review]
Some minor comments after 6/10 code review.

File: spras/local_neighborhood.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        Method to running LocalNeighborhood correctly
+        @param nodes:  input node types with sources and targets (required)
+        @param network:  input network file (required)
+        @param output_file: path to the output pathway file (required)
+        '''
+        print('Running!!!')
+        if not nodes or not network or not output_file:
+            raise ValueError('Required LocalNeighborhood arguments are missing')
+
+        work_dir = '/spras'
```
[the remark this answers is not in the document]
Comment (gabeah) [reply spras/local_neighborhood.py:70]: keeping as-is until we get an answer from @agitter

[the remark this answers is not in the document]
Comment (annaritz) [reply spras/local_neighborhood.py:70]: This is fine - something to clarify when we modify the Contributing Guide text.

**Review by annaritz — APPROVED** [review]

**Comment by annaritz** [comment]
Looks good @gabeah - will close this as done.

**Pull request closed without merging**